pkgs/tools/admin/salt/default.nix

```
@@ -29,6 +30,10 @@ python3.pkgs.buildPythonApplication rec {
 
   patches = [
     ./fix-libcrypto-loading.patch
+    (fetchpatch {
+      url = "https://github.com/saltstack/salt/commit/6ec8b90e402ff3fa8f27c7da70ece898592416bc.patch";
+      sha256 = "a82dcec84d7f2eb5b136ef8112397a1dc1234392741dc6a6db66c868c495eb76";
```

```suggestion
      hash = "sha256-OQCJeG12cp2EZ0BErp6yqsqhv023923rVFDHAFUfF6c=";
```

> I *think* I did it, but I have no idea what I'm doing, nor how to test it.

Looks mostly good.
You can build salt to test it. I.e. in your nixpkgs checkout, run `nix-build -A salt` (or `nix build .#salt` for the flake way, but that needs longer to start the build).
Trying to build, it complains about a wrong hash for me and prints the correct one.

Also, I think using `hash = …` is the more modern variant than `sha256 = …`, but I’m not too up-to-date with all of that.

---

I have no idea, the first and only place I've looked at is https://nixos.wiki/wiki/Nix_Hash
Changed, seems to build for me.